packages/eas-cli/src/credentials/manager/ManageAndroid.ts

```diff
@@ -131,6 +132,18 @@ export class ManageAndroid {
 
         await this.runProjectSpecificActionAsync(ctx, chosenAction, gradleContext);
       } catch (err) {
+        if (
+          err instanceof Error &&
+          err.message ===
+            `Specify "android.package" in ${getProjectConfigDescription(
+              ctx.projectDir
+            )} and run this command again.`
```

Comment:
Is there any better way to determine if this is this specific error (not relying on a message)? 🤔 Ideally, it would be good to use an error code or sth like this.

What do you think?

Reply:
Thought about it, I can make a custom error class for that, probably the best option